sources/representation/qt/mainform.cpp

```diff
@@ -126,6 +126,12 @@ void MainForm::startGameLoop(int id, QString map)
         NODRAW = true;
     }
 
+    if (GetParamsHolder().GetParamBool("-max_fps"))
+    {
+        cap = GetParamsHolder().GetParam<int>("-max_fps");
```

Comment:
If `-max_fps` param is not defined then `cap` will be uninitialized variable and that's bad (any value could be in the `cap`).
You should assign -1 to cap by defult in the main form constructor.  That also will mean that the `-map_fps` is not defined.